manifests/base/crds/workflow-crd.yaml

```diff
@@ -3,6 +3,21 @@ kind: CustomResourceDefinition
 metadata:
   name: workflows.argoproj.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.phase
+    description: Status of the workflow
+    name: Status
+    type: string
+  - JSONPath: .status.startedAt
+    description: When the workflow was started
+    name: Started
+    format: date-time
+    type: date
+  - JSONPath: .status.finishedAt
+    description: When the workflow was finished
+    name: Finished
+    format: date-time
+    type: date
```

Comment:
Instead of `Started` and `Finished` we should try to mimic what `argo list` has:
```sh
$ argo list
NAME            STATUS      AGE   DURATION   PRIORITY
cron-hw-qrjxz   Running     3m    3m         0
cron-hw-kdvsv   Succeeded   9m    5m         0
```
`Age` and `Duration` convey the same information as `Started` and `Finished`, but they fit the K8s pattern more and they will be filled for all Workflows regardless of their Status.

Reply:
`Finished` is not the same as duration, it is how long ago the workflow finished. So in the way I display the information now duration is `Started - Finished`. I don't think it is possible to calculate the duration with `additionalPrinterColumns`, all we can do is select data from the workflow and choose how to format them (a bit). And I did not find `duration` in the workflow resource. 

It might be confusing for people that `kubectl get workflows` and `argo list` will display different, but quite similar-looking, information. So an alternative is to just display them as the strings they are, then it will look like this instead:
```
NAME                         STATUS      STARTED                    FINISHED
abc-87-1573741887810-4nhds   Succeeded   2019-11-13T17:12:44Z       2019-11-13T17:56:17Z"
```

But then it is much harder to calculate the duration in your head by a quick glance. 

Or maybe we just do Age/Started and Status?

Reply:
> I don't think it is possible to calculate the duration with additionalPrinterColumns, all we can do is select data from the workflow and choose how to format them (a bit). 

Yup that's what I figured :)

I think `NAME STATUS AGE` is fine. If users need more information they can simply do a `kubectl describe wf`.